src/components/breadcrumbs/breadcrumbs.js

```diff
@@ -30,20 +33,58 @@ const limitBreadcrumbs = (breadcrumbs, max) => {
     }
   }
 
+  const EuiBreadcrumbCollapsed = () => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+    const ellipsisButton = (
+      <EuiI18n token="euiBreadcrumbs.ariaLabel" default="Show all breadcrumbs">
```

Comment:
I'd be more specific here
```suggestion
      <EuiI18n token="euiBreadcrumbs.collapsedBadge.ariaLabel" default="Show all breadcrumbs">
```